docs/flow/modin/experimental/range_partitioning_groupby.rst

```diff
@@ -1,107 +1,8 @@
-Range-partitioning GroupBy
-""""""""""""""""""""""""""
+:orphan:
```

Comment:
we still may have a reference to this page somewhere out of our docs (messages, blog posts, in-code comments). So I would prefer an automatic redirect to a new page instead of 404